phpcs.xml

```diff
@@ -50,6 +50,11 @@
 		<exclude-pattern>tests/</exclude-pattern>
 	</rule>
 
+	<!-- ignore wp_remote_get in test -->
+	<rule ref="WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents">
+		<exclude-pattern>tests/src</exclude-pattern>
```

Comment:
Excluded this rule because the file I'm getting is not remote and this is a test where we can do what we want with loading files and fixtures.